platform/admin/core/admin-core-insecuredefaults/pom.xml

```diff
@@ -118,17 +118,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.84</minimum>
+                                            <minimum>0.80</minimum>
```

Comment:
I just want to note that these are being dropped because I saw the Jacoco checks failing inconsistently on Windows 2012 when heroing a related PR.